adapters/repos/db/inverted/objects.go

```diff
@@ -94,6 +105,36 @@ func (a *Analyzer) analyzeIDProp(id strfmt.UUID) (*Property, error) {
 	}, nil
 }
 
+func (a *Analyzer) analyzeTimestampProps(input map[string]interface{}) ([]Property, error) {
+	createTime, createTimeOK := input[traverser.InternalPropCreationTimeUnix]
+	updateTime, updateTimeOK := input[traverser.InternalPropLastUpdateTimeUnix]
+
+	var props []Property
+	if createTimeOK {
```

Comment:
just a suggestion: you could create a general method for parsing the timestamp properties from input. Code in `if createTimeOK {}` and `if updateTimeOK {}` looks the same.